Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/OnboardingDocumentConfig.hs

```diff
@@ -0,0 +1,50 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
+
+module Storage.CachedQueries.OnboardingDocumentConfig
+  ( findByMerchantIdAndDocumentType,
+    clearCache,
+  )
+where
+
+import Domain.Types.Merchant (Merchant)
+import Domain.Types.OnboardingDocumentConfig
+import Kernel.Prelude
+import qualified Kernel.Storage.Hedis as Hedis
+import Kernel.Types.Id
+import Kernel.Utils.Common
+import Storage.CachedQueries.CacheConfig
+import qualified Storage.Queries.OnboardingDocumentConfig as Queries
+
+
+findByMerchantIdAndDocumentType :: (CacheFlow m r, EsqDBFlow m r) => Id Merchant -> DocumentType -> m (Maybe OnboardingDocumentConfig)
```

Comment:
I think you need to delete the onboarding config from dhall, and use this function where it is needed.

Reply:
yes that will be done in the second part.